client/src/components/IconComponent.jsx

```diff
@@ -43,13 +58,25 @@ class IconComponent extends React.Component {
 
   render() {
     const { icons } = this.props;
+    const { currentIndex } = this.state;
     if (icons && icons.length > 0) {
       const displayIcons = icons.map((icon, index) => {
         return (
-          <StyledIconButton key={icon.icon_title} onClick={() => this.updateSelectedIcon(index)}>
-            <svg fill="white" width="50" height="50" viewBox="0 0 32 32" xmlns="http://www.w3.org/2000/svg" preserveAspectRatio="xMidYMid meet">
+          <StyledIconButton
+            key={icon.icon_title}
+            onClick={() => this.updateSelectedIcon(index)}
+            isSelected={currentIndex === index}
+          >
+            <StyledIconSVG
+              isSelected={currentIndex === index}
```

Comment:
I like this

Reply:
@danielharmon I was looking through your code to see how you  were incorporating more dynamic styling and learned a few things, one of which was passing props into a styled component and changing the appearance based on a conditional!